todo_app/tests/test_api.py

```diff
@@ -0,0 +1,55 @@
+from dotenv import find_dotenv , load_dotenv
+from todo_app import app
+from mock import patch, Mock
+import os
+import pytest
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    print(file_path)
+    load_dotenv(file_path, override=True)
+    # Create the new app.
+    test_app = app.create_app()
+     # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+        
+@patch('requests.get')
+def test_index_page(mock_get_requests, client):
+    mock_get_requests.side_effect = mock_get_lists
+    response = client.get('/')
+
+    assert response.status_code == 200
+
+@patch('requests.put')
+def test_add_page(mock_get_requests, client):
```

Comment:
In this test we don't actually trigger any API calls, so we could remove the patch here (and note that we'd actually be patching `PUT` calls, but we'd be trying to match `GET` requests

Reply:
Hi Jack, 
Not sure I understand this. If I remove the patch it fails.